internal/scorecard/alpha/scorecard.go

```diff
@@ -105,25 +104,68 @@ func (o Scorecard) selectTests() []Test {
 	return selected
 }
 
-// runTest executes a single test
-func (o Scorecard) runTest(ctx context.Context, test Test) (result v1alpha2.ScorecardTestResult, err error) {
+func (r FakeTestRunner) Initialize(ctx context.Context) (configMapName string, err error) {
+	return configMapName, nil
+}
+
+// Initialize sets up the bundle configmap for tests
+func (r PodTestRunner) Initialize(ctx context.Context) (configMapName string, err error) {
```

Comment:
Make this a pointer receiver so we can set fields on `r` that persist after the function returns.
```suggestion
func (r *PodTestRunner) Initialize(ctx context.Context) (configMapName string, err error) {
```

Reply:
changed to add the pointer receiver in latest commit.